doc/userguide/install.rst

```diff
@@ -140,34 +113,59 @@ one of the following ways::
 
 Minimal::
 
-    # Installed Rust and cargo as indicated above
+    sudo dnf install -y rustc cargo
+    cargo install --force cbindgen
+    # Make sure the cargo path is within your PATH environment e.g.:
+    echo 'export PATH=”${PATH}:~/.cargo/bin”' >> ~/.bashrc
+    export PATH="${PATH}:~/.cargo/bin"
     sudo dnf install -y gcc gcc-c++ git jansson-devel libpcap-devel libtool \
                    libyaml-devel make pcre2-devel which zlib-devel
-    cargo install --force cbindgen
 
 Recommended::
 
-    # Installed Rust and cargo as indicated above
-    sudo dnf install -y autoconf automake diffutils file-devel gcc gcc-c++ git \
-                   jansson-devel jq libcap-ng-devel libevent-devel \
-                   libmaxminddb-devel libnet-devel libnetfilter_queue-devel \
-                   libnfnetlink-devel libpcap-devel libtool libyaml-devel \
-                   lua-devel lz4-devel make nss-devel pcre2-devel pkgconfig \
-                   python3-devel python3-sphinx python3-yaml sudo which \
-                   zlib-devel
-    cargo install --force cbindgen
+    # Minimal dependencies installed and then
+    sudo dnf install -y epel-release
+    sudo dnf install -y autoconf automake clang diffutils file-devel \
+                    hiredis-devel hyperscan-devel \
+                    jansson-devel jq libbpf-devel libcap-ng-devel \
+                    libevent-devel libmaxminddb-devel libnet-devel \
+                    libnetfilter_queue-devel libnfnetlink-devel libpcap-devel \
+                    libtool libyaml-devel llvm-toolset lua-devel \
+                    lz4-devel make nspr-devel nss-devel pcre2-devel \
+                    pkgconfig python3-devel python3-sphinx python3-yaml \
+                    sudo which zlib-devel
 
 Compilation
-^^^^^^^^^^^
+"""""""""""
 
 Follow these steps from your Suricata directory::
 
-    ./scripts/bundle.sh
-    ./autogen.sh
     ./configure # you may want to add additional parameters here
     # ./configure --help to get all available parameters
-    make -j8 # j is for paralleling, you may de/increase depending on your CPU
+    # Recommended parameters:
+    CC=clang ./configure --enable-ebpf --enable-ebpf-build --enable-nfqueue \
```

Comment:
I don't think we're generally recommending clang over gcc or recommending enabling ebpf and or pretty much any of the other "enable" options. They are disabled by default for a reason

Reply:
I derived these "enable" options from the packages installed in the previous Recommend dependencies section. As a user, I would be clueless as to why the recommended packages are recommended and what to configure as a next step. This configure step enables most of the "good stuff" that Suricata offers. If you disagree, what would you like me to do? 
1) remove altogether
2) Reword "recommended parameters" to "possible configure parameters"

Reply:
@victorjulien do you have any preference? I would personally opt for the second option.

Reply:
I don't think this section should really change.

What would likely be nice is a section on "build options" where these are covered, and what new dependencies they might bring in. A bigger job, but more useful.

Possible example, but a bit verbose: https://nginx.org/en/docs/configure.html

Reply:
Created a ticket: https://redmine.openinfosecfoundation.org/issues/6824